packages/sdk/src/CLRunner.ts

```diff
@@ -1522,9 +1582,39 @@ export class CLRunner {
                     // the copy of map is created because the passed infilledEntityMap contains "filledEntities by Id" too
                     // and this causes issues when calculating changedFilledEntities.
                     const entityMapBeforeCall = new CLM.FilledEntityMap(await state.EntityState.FilledEntityMap())
-                    // Store logic callback value
-                    const logicObject = await callback.logic(memoryManager, ...renderedLogicArgumentValues)
-                    logicResult.logicValue = JSON.stringify(logicObject)
+
+                    // If callback result is set use it
+                    // Otherwise use logic function
+                    let logicReturnValue: unknown
+                    if (actionInput.resultName) {
```

Comment:
As I'm reading this, I thinking "resultName" could actually be a big confusing.   Maybe "mockResultName"?   Matches "mockResults" in Callback.ts

Reply:
changed to `mockResultName`